src/wp-includes/post.php

```diff
@@ -6804,9 +6804,10 @@ function wp_attachment_is_image( $post = null ) {
  * @since 2.1.0
  *
  * @param string|int $mime MIME type or attachment ID.
```

Comment:
```suggestion
 * @param string|int $mime          MIME type or attachment ID.
```

(lining up parameter descriptions)